frontend/src/constants/Routes.ts

```diff
@@ -9,8 +9,43 @@ export const EMAIL_ACTION_PAGE = "/email-action";
 // Private Admin Routes
 export const ADMIN_LANDING_PAGE = "/admin";
 export const USERS_PAGE = "/admin/users";
+
 export const ASSESSMENTS_PAGE = "/admin/assessments";
-export const ASSESSMENT_EDITOR_PAGE = "/admin/assessment-editor";
+export const ASSESSMENT_EDITOR_BASE = ({
+  assessmentId,
+}: {
+  assessmentId?: string;
+}) => "/admin/assessments/" + (assessmentId ?? "create");
+export const ASSESSMENT_EDITOR_PAGE = ({
+  assessmentId,
+}: {
+  assessmentId?: string;
+}) => ASSESSMENT_EDITOR_BASE({ assessmentId }) + (assessmentId ? "/edit" : "");
+export const ASSESSMENT_EDITOR_PREVIEW_PAGE = ({
+  assessmentId,
+}: {
+  assessmentId?: string;
+}) => ASSESSMENT_EDITOR_BASE({ assessmentId }) + "/preview";
+export const ASSESSMENT_EDITOR_QUESTION_EDITOR_PAGE = ({
+  assessmentId,
+  questionIndex,
+}: {
+  assessmentId?: string;
+  questionIndex?: string;
+}) =>
+  ASSESSMENT_EDITOR_BASE({ assessmentId }) +
+  "/questions/" +
+  (questionIndex ? `${questionIndex}/edit` : "add");
```

Comment:
`add` seems a bit weird to me. would `new` be better?

Reply:
Makes no difference to me, I can change it